src/bundle/Resources/encore/ibexa.config.js

```diff
@@ -1,5 +1,8 @@
 const path = require('path');
 
 module.exports = (Encore) => {
+    Encore.addEntry('ibexa-design-system-storybook-js', [
+        path.resolve('./vendor/ibexa/design-system-twig/src/bundle/Resources/public/ts/init_components.ts')]
```

Comment:
I'm not sure if this absolute path is safe. I think the `vendor` directory can be renamed.